x-pack/solutions/observability/plugins/observability/public/pages/alert_details/components/related_dashboards/dashboard_tile.tsx

```diff
@@ -61,7 +66,14 @@ export function DashboardTile({
           <EuiText color={'subdued'} size="s">
             {dashboard.description}
           </EuiText>
-        </EuiFlexItem>
+          {tagsReferences.length ? (
```

Comment:
Can you add a test for DashboardTile that includes multiple tags?

Reply:
Done!